src/error.rs

```diff
@@ -87,6 +90,7 @@ impl IntoResponse for AppError {
             | AppError::JwtVerification
             | AppError::WrongAdminPassword => StatusCode::UNAUTHORIZED,
             AppError::MissingLayerJson => StatusCode::UNPROCESSABLE_ENTITY,
+            AppError::BioTooLong(_) => StatusCode::UNPROCESSABLE_ENTITY,
```

Comment:
```suggestion
            AppError::MissingLayerJson | AppError::BioTooLong(_) => StatusCode::UNPROCESSABLE_ENTITY,
```
you can match multiple variants to single value like so